docs/FAQ.md

```diff
@@ -554,33 +554,6 @@ Usually, the different commits associated with the conflicted bookmark should al
 appear in the log, but if they don't you can use `jj bookmark list`to show all the
 commits associated with it.
 
-### How do I integrate Jujutsu with Gerrit?
```

Comment:
Rather than completely removing this, should we link to gerrit.md?

Reply:
https://github.com/jj-vcs/jj/pull/7546#issuecomment-3308975016

My original version kept a link. Seems we have conflicting opinions. :)

I don’t mind either way.

Reply:
I think it's fine; people can do a quick search and it should show up easily with any search method, so I don't think we need to duplicate the info anywhere.